doc/source/io.rst

```diff
@@ -2059,6 +2064,10 @@ using Hadoop or Spark.
   df
   df.to_json(orient='records', lines=True)
 
+  reader = pd.read_json(StringIO(jsonl), lines=True, chunksize=1)
```

Comment:
maybe a comment here that reader is an iterator

Reply:
done